README.md

```diff
@@ -82,6 +82,8 @@ As a basic example, consider a UI that shows a number along with "+" and "−" b
 To implement this feature we create a new type that will house the domain and behavior of the feature by conforming to `ReducerProtocol`:
 
 ```swift
+import ComposableArchitecture
```

Comment:
I'm not sure this is required here, as what's follow is not intended to be a regular source file (see for example l.106 with `struct State: Equatable { … }`).

Reply:
But if you follow the code samples then it makes sense that the author is just saving space by not repeating and just adding new code to the struct